### PR TITLE
Feature/5597 mp api schema updates

### PR DESCRIPTION
**Pull request by jeffreyaggrey**

### CODE REVIEW

File: src/dtos/monitor-plan-update.dto.ts [hunk 1]
```diff
@@ -30,7 +30,7 @@ export class UpdateMonitorPlanDTO {
 
   @ValidateNested()
   @Type(() => MonitorPlanCommentBaseDTO)
-  comments: MonitorPlanCommentBaseDTO[];
+  monitoringPlanCommentData: MonitorPlanCommentBaseDTO[];
 
   @ValidateNested()
   @Type(() => UnitStackConfigurationBaseDTO)
```
Comment (subrataroyCVP) [inline src/dtos/monitor-plan-update.dto.ts:36]: missed to update `unitStackConfigurations` and `locations`

**Review by subrataroyCVP — CHANGES_REQUESTED** [review]
`unitOfStandard` need to be updated to `unitofStandard`.

File: src/component-workspace/component.service.ts [hunk 2]
```diff
@@ -22,7 +22,7 @@ export class ComponentWorkspaceService {
     private readonly logger: Logger,
 
     @Inject(forwardRef(() => AnalyzerRangeWorkspaceService))
-    private readonly analyzerRangeService: AnalyzerRangeWorkspaceService,
+    private readonly analyzerRangeDataervice: AnalyzerRangeWorkspaceService,
```
Comment (billmiller13) [inline src/component-workspace/component.service.ts]: Misspelled analyzerRangeDataService

**Review by billmiller13 — COMMENTED** [review]
Looks good except for the one comment I left where you misspelled something

**Review by subrataroyCVP — APPROVED** [review]
All the DTO changes looks good. Tested with few monitorPlans, Import and Export working as expected.

**Comment by subrataroyCVP** [comment]
@fanwari1 Merging the PR. Let @jeffreyaggrey know about the changes, if any.

**Pull request merged**